src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -64,17 +74,18 @@ def consume_scenes(scene_list: list) -> list:
     :return: scene_data
     """
     scene_data = []
+    errors = []
 
     for scene in scene_list:
         current_scene_data = {
             'data': []
         }
 
         if 'scene_id' not in scene:
-            raise ValueError('Every scene must have a scene ID.')
+            errors.append(f'Every scene must have a scene ID')
 
         if 'start' not in scene: 
-            raise ValueError('You must specify a starting time for a scene.')
+            errors.append(f'    does not have a start time')
```

Comment:
Some strange spacing here. I don't remember if start time needs a required value but we aren't enforcing it here or in the start time validator. As a result it's not showing in the ui

Reply:
The spacing is a visual indicator. When you get an error, you will see 4 spaces if it's a scene error, and 8 spaces if it's a caption error, so it will look something like this
There is a error in scene id 5
    there is no start time
    there is an error in caption id 1
        there is no caption string
        color rust is not supported by foreground
    there is an error in caption id 4
        column 88 is not supported

and so on.

Reply:
it doesn't show in the ui, maybe `\t` will work

Reply:
Oh that's weird, it did when I tested it with chrome on windows. You use mac right? I'll make sure to try tabs, we want it to look right on macs.

Reply:
Fixed spaces with tabs